BigBaseV2/src/core/globals.hpp

```diff
@@ -242,21 +242,22 @@ namespace big
 
 			AutoDriveDestination auto_drive_destination = AutoDriveDestination::STOPPED;
 			AutoDriveStyle auto_drive_style = AutoDriveStyle::LAW_ABIDING;
+			float auto_drive_speed = 1;
 			bool auto_turn_signals = false;
+			int boost_behavior = 0;
```

Comment:
`boost_behavior` should be changed to an enum to prevent invalid values from being assigned and removes the obscure meaning behind `0/1/2`.

Reply:
BoOsT_BeHaViOr ShOuLd bE ChAnGeD To aN EnUm tO PrEvEnT InVaLiD VaLuEs fRoM BeInG AsSiGnEd aNd rEmOvEs tHe oBsCuRe mEaNiNg bEhInD 0/1/2.